_posts/2017/12/2017-12-08-cc-poster-comp.md

```diff
@@ -0,0 +1,81 @@
+---
+### Design a Promotional Poster for CarpentryCon 2018
+
+The Carpentries are excited to announce our poster design competition where some lucky member of our community will win a **free** registration for [CarpentryCon 2018](https://www.carpentrycon.org/).
+
+Are there any artists out there who could design a fantastic poster to promote CarpentryCon 2018? If so, we want to see your design!
+
+The winning designer will gain free registration to CarpentryCon 2018. This includes registration for the conference only. It does not include travel to and from Dublin.
+
+#### Information for the Poster
+
+CarpentryCon 2018 will take place in Dublin from 30 May - 1 June, 2018 at [University College Dublin](http://www.ucd.ie/) with the overarching theme of *Building Globally, Connecting Locally*.
+
+The three-day event will focus on professional development, community building and networking, as we aim to help 
+skill up the next generation of research leaders. Find out more from the [CarpentryCon 2018 website](http://www.carpentrycon.org/).
+
+#### About the Competition
+
+<table>
+<tr>
+<td valign="top" width="30%">What will the poster be used for?</td>
+<td valign="top">The winning poster will be used for promotional and advertising purposes and featured in all CarpentryCon 2018 
+advertising across a range of social media, including <a href="https://twitter.com/swcarpentry">Twitter</a> and <a href="https://www.facebook.com/carpentries">Facebook</a>.</td>
+</tr>
+<tr>
+<td valign="top" width="30%">When can you submit a design?
+</td>
+<td valign="top">Entries can be submitted now.</td>
+</tr>
+
+<tr>
+<td valign="top" width="30%">What is the closing date for entries?</td>
+<td valign="top">5pm UTC on 31 January, 2018. We will announce the winner on 10 February, 2018.</td>
+</tr>
+
+<tr>
+<td valign="top" width="30%">Who can enter?</td>
+<td valign="top">Anyone may enter, though entrants must be over the age of 18. </td>
+</tr>
+
+<tr>
+<td valign="top" width="30%">Can I submit more than one design?</td>
+<td valign="top">Yes, competition entrants are welcome to submit as many designs as they like.</td>
+</tr>
+
+<tr>
+<td valign="top" width="30%">What does the poster need to include?</td>
+<td valign="top">Posters should reflect the conference theme of <em>Building Globally, Connecting Locally</em>, and should also include the venue and dates (see above) and a link to the <a href="http://www.carpentrycon.org/">CarpentryCon website</a>.
+</td>
+</tr>
+
+<tr>
+<td valign="top" width="30%">How will the judging be done?</td>
+<td valign="top">Entries in the poster competition will be assessed by the members of the CarpentryCon task force who will choose the poster that best embodies the conference theme of <em>Building Globally, Connecting Locally</em>. The winner will be notified by email once the decision has been reached, and unsuccessful entrants will also be notified by email of the result. </td>
+</tr>
```

Comment:
Same here about the theme. Need to decide if it's this or "Building Locally, Connecting Globally"